topic-controller/src/main/java/io/strimzi/controller/topic/BaseKafkaImpl.java

```diff
@@ -31,7 +31,7 @@
  */
 public abstract class BaseKafkaImpl implements Kafka {
 
-    private final static Logger logger = LoggerFactory.getLogger(BaseKafkaImpl.class);
+    private final static Logger LOGGER = LoggerFactory.getLogger(BaseKafkaImpl.class);
```

Comment:
does it force to have "final static" always uppercase ? :-(

Reply:
A `static final` is a constant and the Sun Java convention recommended CAPS for those since as long as I can remember.